src/everest/plugins/hook_impl.py

```diff
@@ -58,3 +58,8 @@ def get_forward_models_schemas():
 @hookimpl
 def installable_workflow_jobs():
     return None
+
+
+@hookimpl
+def get_forward_model_documentations():
```

Comment:
Is this needed? I see there are similar entries in there from before, and wondering the same thing about them (just relevant, but outside scope of PR)

Reply:
I do not think the other functions in this interface provides what we want and I don't want to change existing behavior. Or what were you thinking about?